build/common/installer/scripts/tomlparser-common-agent-config.rb

```diff
@@ -9,6 +9,7 @@
 @configSchemaVersion = ""
 
 @disableTelemetry = false
+@logEnableKubernetesMetadataCacheTTLSeconds = 60
```

Comment:
is this default used when there is no configmap as well?

Reply:
when there is no config map, this feild will not be used